core/Updates/4.0.0-b1.php

```diff
@@ -53,9 +54,6 @@ public function getMigrations(Updater $updater)
         $columnsToAdd = [];
 
         $migrations = [];
-        $migrations[] = $this->migration->db->changeColumnType('log_action', 'name', 'VARCHAR(4096)');
```

Comment:
we should avoid slow updates in the beginning and have some required faster ones first